modules/server/src/main/scala/navigate/server/tcs/TcsBaseControllerEpics.scala

```diff
@@ -263,32 +272,41 @@ class TcsBaseControllerEpics[F[_]: Async: Parallel: Temporal](
                 )
               )
           )
-          .getOrElse(identity[TcsCommands[F]])
+          .getOrElse(
+            setProbeTracking(
+              Getter[TcsCommands[F], ProbeTrackingCommand[F, TcsCommands[F]]](
+                _.oiwfsProbeTrackingCommand
+              ),
+              TrackingConfig.noTracking
+            )
+          )
       )
 
   // Added a 1.5 s wait between selecting the OIWFS and setting targets, to copy TCC
   override def tcsConfig(config: TcsBaseController.TcsConfig): F[ApplyCommandResult] =
-    (
-      selectOiwfs(config) *>
-        VerifiedEpics.liftF(Temporal[F].sleep(1500.milliseconds)) *>
-        applyTcsConfig(config)(
-          sys.tcsEpics.startCommand(timeout)
-        ).post
-    ).verifiedRun(ConnectionTimeout)
+    disableGuide *>
+      (
+        selectOiwfs(config) *>
+          VerifiedEpics.liftF(Temporal[F].sleep(1500.milliseconds)) *>
+          applyTcsConfig(config)(
+            sys.tcsEpics.startCommand(timeout)
+          ).post
+      ).verifiedRun(ConnectionTimeout)
 
   override def slew(
     slewOptions: SlewOptions,
     tcsConfig:   TcsBaseController.TcsConfig
   ): F[ApplyCommandResult] =
-    (
-      selectOiwfs(tcsConfig) *>
-        VerifiedEpics.liftF(Temporal[F].sleep(1500.milliseconds)) *>
-        setSlewOptions(slewOptions)
-          .compose(applyTcsConfig(tcsConfig))(
-            sys.tcsEpics.startCommand(timeout)
-          )
-          .post
-    ).verifiedRun(ConnectionTimeout)
+    disableGuide *>
+      (
+        selectOiwfs(tcsConfig) *>
+          VerifiedEpics.liftF(Temporal[F].sleep(1500.milliseconds)) *>
```

Comment:
How's that wait time defined? Should it be configurable?

Reply:
Nah, I just copied what works in TCC. I could put it in a variable if that makes you happy/